Apps/Src/FaultState.c

```diff
@@ -62,6 +62,7 @@ void EnterFaultState(void) {
             } else {
                 hall_fault_cnt++;
                 MotorController_Restart(); //re-initialize motor
+                FaultBitmap = FAULT_NONE; // Clearing entire bitmap because it can only hold one error
```

Comment:
Same thing here as the other comment. We only need to clear one bit as opposed to the entire bitmap.